data/js/server/teleporter/teleporter_solen_hive.js

```diff
@@ -0,0 +1,332 @@
+/*
+*	Solen Hives and Ants was introduced in LBR during a event called When Ants Attack.
+*
+*	We have the Randomly spawning black holes that can be put out with event tags for events.
+*	Description how it works
+*	1. When walked on it will spawn 2 to 5 ants
+*	2. Keeps track of the deaths of each ant so it cant over spawn the area.
+*	3. Has a 5 min cool down so it can't keep spawning when walking over.
+*	4. Double click on the hole and it will take you to the solen hive.
+*
+*	We also have the Main Solen hive Entrances.
+*	Description how it works
+*	1. You double click it will take you down to the solen hive depending on what location you are using.
+*	2. Double click in the middle of the black inside the solen hive it will return you to the surface.
+*
+*	World Decorations that come with this file will place the Main Entrances and Exits around the World for Felucca Map only.
+*/
+
+// Entrance coordinates
+var entranceMap = {
+	// Minoc
+	"2608,763": [5916, 1793],
+	"2609,763": [5921, 1793],
+	"2608,764": [5917, 1797],
+	"2609,764": [5921, 1797],
+	// Trinsic
+	"1690,2789": [5653, 2018],
+	"1691,2789": [5658, 2017],
+	"1690,2790": [5655, 2021],
+	"1691,2790": [5658, 2021],
+	// Compassion Shrine
+	"1724,814": [5919, 2019],
+	"1725,814": [5922, 2020],
+	"1724,815": [5917, 2023],
+	"1725,815": [5922, 2024],
+	// Yew Orc Fort
+	"730,1451": [5654, 1789],
+	"731,1451": [5658, 1791],
+	"730,1452": [5652, 1793],
+	"731,1452": [5656, 1795],
+
+	// Inside Solen Hive Teleporters
+	//Area A
+	//B leads to level 2
+	"5757,1794": [5725, 1894],
+	"5755,1795": [5725, 1894],
+	"5756,1794": [5725, 1894],
+	"5757,1796": [5725, 1894],
+	"5757,1795": [5725, 1894],
+	"5758,1795": [5725, 1894],
+
+	//B leads back level 1
+	"5727,1894": [5757, 1798],
+	"5728,1896": [5757, 1798],
+	"5728,1895": [5757, 1798],
+	"5728,1894": [5757, 1798],
+
+	//C leads to center area
+	"5657,1898": [5750, 1894],
```

Comment:
So this is "C leads to center area", but 2 above with the same target coordinates is "B leads to level 2"

Are "center area" and "level 2" the same?